docs/concepts/models/model_kinds.md

```diff
@@ -288,7 +288,6 @@ MODEL (
 
 * BigQuery
 * Databricks
-* Postgres
```

Comment:
Thanks for catching this.

Reply:
Wasn't me unfortunately :) This was reported by a user in Slack